from app.core.exceptions.company.company import CompanyNotFoundException
from app.core.exceptions.company.department import DepartmentNotFoundException